### PR TITLE
Add AddWorkExperienceCommand

### LINKED ISSUE #72: Add Work Experience Functionality
due:Thursday

### PR DESCRIPTION
**Pull request by kuiktzecheng**
Add a new command "addw" for users to add work experience to existing contact in the contact list. Fixes #72

### CODE REVIEW

**Review by verakohh — APPROVED** [review]
lgtm!

File: src/main/java/seedu/address/logic/parser/AddWorkExperienceCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        for (String arg : splitArgs) {
+            if (arg.startsWith("in/")) {
+                indexString = arg.substring(3); // Remove "in/" prefix
+            } else if (arg.startsWith("w/")) {
+                workExperienceString = arg.substring(2); // Remove "w/" prefix
+            }
+        }
+
+        if (indexString == null || workExperienceString == null || workExperienceString.isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
```
Comment (jeremychiaaaaa) [inline src/main/java/seedu/address/logic/parser/AddWorkExperienceCommandParser.java:39]: Please add in test for this block of code

File: src/main/java/seedu/address/logic/parser/AddWorkExperienceCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            Index index = ParserUtil.parseIndex(indexString);
+
+            // Parse the WorkExp object
+            WorkExp workExp = ParserUtil.parseWorkExp(workExperienceString);
+
+            // Return the new AddWorkExperience command
+            return new AddWorkExperienceCommand(index, workExp);
+
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
```
Comment (jeremychiaaaaa) [inline src/main/java/seedu/address/logic/parser/AddWorkExperienceCommandParser.java:54]: Please end in tests for this code to make sure the exception is thrown when an invalid command format is passed

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 3]
```diff
@@ -59,6 +60,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddCommand.COMMAND_WORD:
             return new AddCommandParser().parse(arguments);
 
+        case AddWorkExperienceCommand.COMMAND_WORD:
+            return new AddWorkExperienceCommandParser().parse(arguments);
```
Comment (jeremychiaaaaa) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:64]: Add in a test case that checks whether the parser can detect the new command word added

**Review by jeremychiaaaaa — APPROVED** [review]
LGTM

**Review by verakohh — COMMENTED** [review]
lgtm, i think you can include more tests

**Pull request merged**